### ISSUE TITLE
Is it possible to use the / operator for dealing with nested enums?

### ISSUE THREAD

**Issue by amarantedaniel** [pointfreeco/swift-case-paths#12]
Let's say I have these three enums:

```swift
enum First {
    case second(Second)
}

enum Second {
    case third(Third)
}

enum Third {
    case hello
}
```

I know that I can create a case path from first to second like this:
```swift
let casePath: CasePath<First, Second> = /First.second
```

And I also know that I can create a case path from first to third using the constructor like this:
```swift
let nestedCasePath = CasePath(
    embed: { third in
        First.second(Second.third(third))
    }, extract: { first in
        if case let First.second(Second.third(third)) = first {
            return third
        }
        return nil
    }
)
```

What I'd like to know is if it's possible to use the `/`  operator to deal with these nested case paths, or if the only way is to create them using the constructor.

Thanks.

**Comment by stephencelis** [comment]
Hey @amarantedaniel! Absolutely. The README has an example of using the `..` operator for composition:

```swift
/Result<Authentication, Error>..Authentication.authenticated
// CasePath<Result<Authentication, Error>, String>
```

So in your case you can do:

```swift
/First.second..Second.third
```

**Comment by amarantedaniel** [comment]
Thank you @stephencelis .

**Comment by amarantedaniel** [comment]
Hmm I just noticed something.

I'm using case paths with the ComposableArchitecture.

`/First.second..Second.third ` will only compile if I import `CasePaths`.

Everything else works without importing it. Probably because it's made available through ComposableArchitecture

**Comment by stephencelis** [comment]
Ah bummer. That might be a bug with `@_exported import`, though because it's not a public API I don't know if we can really file a bug against it...

**Comment by amarantedaniel** [comment]
That's ok, but It's probably why I couldn't figure out how to do it before.

**Comment by maximkrouk** [comment]
Seems like it was fixed, can't reproduce it in Xcode 13.2.1
> Btw, I also have a separate module that exports TCA, so even nesting exports should work now
